### PR TITLE
Update README.md about the cross compile

### PR DESCRIPTION
**Pull request by lzbj**
with Docker/gradle.

Address issue #197.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,29 +92,26 @@ $ git branch --set-upstream-to origin/master  # track master from origin now
 
 You can now use `git push` to push changes to your repository and submit pull requests.
 
-# How to Build with Docker
-If you don't want to bother with go installation, build, git clone etc, you can do it with Docker, then
-you can run wskdeploy tool in your container.
+# How to Cross Compile Binary with Gradle/Docker
+If you don't want to bother with go installation, build, git clone etc, and you can do it with Gradle/Docker.
+After compile, suitable wskdeploy binary that works for your OS platform will be available under /bin directory.
```
Comment (paulcastro) [inline README.md]: Warning: nit comment, add an "a":
`After compile, a suitable wsdeploy binary .....`

File: README.md [hunk 2]
```diff
@@ -92,29 +92,26 @@ $ git branch --set-upstream-to origin/master  # track master from origin now
...
-docker build -f Dockerfile .  -t openwhisk/wskdeploy
-```
+3. Clone the wskdeploy repo with command ```git clone https://github.com/openwhisk/openwhisk-wskdeploy.git```
 
-4. Bring up the docker container.
-```
-docker run -ti openwhisk/wskdeploy
-```
-5. Inside the container, run `wskdeploy` and have fun.
+3. If you use Windows OS, type ```gradlew.bat -version ```, for Unix/Linux/Mac, please type ```./gradlew -version```.
```
Comment (paulcastro) [inline README.md]: f you use Windows OS, type ```gradlew.bat -version ```. For Unix/Linux/Mac, please type ```./gradlew -version```.

File: README.md [hunk 3]
```diff
@@ -92,29 +92,26 @@ $ git branch --set-upstream-to origin/master  # track master from origin now
...
+
+4. Make sure you can see the correct Gradle version info on your console. Currently the expected Gradle
+version is 3.3.
+
+5. For Windows type ```gradlew.bat distDocker```, for Linux/Unix/Mac, please type ```./gradlew distDocker```. These
```
Comment (paulcastro) [inline README.md]: For Windows type ```gradlew.bat distDocker```. For Linux/Unix/Mac, please type ```./gradlew distDocker```.

File: README.md [hunk 4]
```diff
@@ -92,29 +92,26 @@ $ git branch --set-upstream-to origin/master  # track master from origin now
...
+commands will start the wskdeploy cross compile for your specific OS platform inside a Docker container.
+
+6. After build success, you could find a correct binary under current /bin dir of you openwhisk-deploy clone dir.
```
Comment (paulcastro) [inline README.md]: `After build success, you should ... `

**Comment by csantanapr** [comment]
@lzbj Can you address comments from @paulcastro

**Comment by lzbj** [comment]
Yes, done.

**Comment by lzbj** [comment]
@daisy-ycguo , please help merge if you could, as I addressed the comments already, or I have to rebase again.

**Pull request merged**